src/lib.rs

```diff
@@ -100,6 +103,21 @@ pub use crate::{
     uint::{UInt, UTerm},
 };
 
+#[doc(no_inline)]
+#[rustfmt::skip]
+pub use consts::{
+    False, True, B0, B1,
+    U0, U1, U2, *,
+    N1, N2, Z0, P1, P2, *,
```

Comment:
Nit: re-export the negative numbers in the right order:

```suggestion
    N2, N1, Z0, P1, P2, *,
```